### PR TITLE
add cvmfs_rsync script to cvmfs-server package

### PR DESCRIPTION
**Pull request by DrDaveD**
As discussed in https://sft.its.cern.ch/jira/browse/CVM-814

### CODE REVIEW

**Comment by DrDaveD** [comment]
Please don't forget this pull request

File: cvmfs/cvmfs_rsync [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Rsync directories into cvmfs.
+
+ME=cvmfsrsync
```
Comment (jblomer) [inline cvmfs/cvmfs_rsync]: This should be `cvmfs_rsync`, no?

**Comment by jblomer** [comment]
Apologies for the delay.  Looks fine to me.  @reneme any comments? 

Can you add a few integration tests for this new tool?

**Comment by reneme** [comment]
Why doesn't the script work with remote hosts? Is there a technical reason? I'm not sure if it is an operational problem, just curious.

File: cvmfs/cvmfs_rsync (hunk 1 above)
Comment (DrDaveD) [inline cvmfs/cvmfs_rsync]: You're right, I corrected it to cvmfs_rsync

**Comment by DrDaveD** [comment]
> Can you add a few integration tests for this new tool?
Is it alright if I do that in a later pull request?  I have other priorities at the moment and we're in conversation about a possible alternative that would make this obsolete anyway.

> Why doesn't the script work with remote hosts? Is there a technical reason? 
Yes, it needs to look for missing directories in the source and .catalog files in the destination and it's not practical to do either remotely.  This limitation is one of the reasons (along with the script's general complexity) that I asked in the ticket https://sft.its.cern.ch/jira/browse/CVM-814 about the possibility of an alternative that bypasses the intermediate step of creating .cvmfscatalog files from .cvmfsdirtab, going instead directly from .cvmfsdirtab to catalog.  (Please respond to the discussion there because I think it will be easier to locate in the future than this pull request).

**Comment by jblomer** [comment]
If you need this urgently merged, I can do it of course.  Otherwise, I wouldn't mind leaving the pull request open for amendment.

**Comment by DrDaveD** [comment]
It's not that it's urgently needed, but it would be helpful for it to be in a place that I could easily refer people to the latest version.  Also in general it is a pain to let new code branches languish as common files (in this case the cmake file and packaging descriptors) get modified by other updates and need later to be merged.  Frequently merging smaller working pieces into the common branch is the easiest way to do joint development.

**Comment by DrDaveD** [comment]
I want to move this out of my devel branch so I'm closing this request and I'll open a new one

**Pull request closed without merging**